components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtilsTest.java

```diff
@@ -88,11 +130,160 @@ public void testGetAppAuthenticatorByNameValidAuthenticator(){
     }
 
     @Test
-    public void testGetAppAuthenticatorByNameNonExistAuthenticator(){
+    public void testGetAppAuthenticatorByNameNonExistAuthenticator() {
```

Comment:
Add new line after the method signature

Reply:
Already fixed with commit baf49d4b0e85432467ae5848dcf443c0032ce5c9 in https://github.com/wso2/carbon-identity-framework/pull/3509 PR